bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd

```diff
@@ -52,6 +52,19 @@
         </xsd:complexContent>
     </xsd:complexType>
 
+    <xsd:complexType name="Facility">
+        <xsd:sequence>
+            <xsd:element name="name" type="xsd:string" minOccurs="0" maxOccurs="1"/>
+            <xsd:element name="aeonCompatible" type="xsd:boolean" minOccurs="0" maxOccurs="1"/>
+        </xsd:sequence>
+    </xsd:complexType>
```

Comment:
I'm confused by what feeds the `aeonCompatible` element.  I mean, how does that get set in a proposal document.

Reply:
There is a UI (still pending) that will let you select from a number of facilities and potentially the time requested at each